### ISSUE TITLE
Ctl+Z doesn't work

### ISSUE THREAD

**Issue by LiraBrik** [excalidraw/excalidraw#6194]
Thanks for project. It's just cool! Ctl+Z doesn't work on russian keyboard layout. Аfter the hotkeys, the "Step back" button in the lower left corner also stops working.

**Comment by dwelle** [comment]
Hi. I can reproduce the shortcut not working on a russian keyboard layout, but can't reproduce the button (which are also not tied to the keyboard in away) 🤔

**Comment by LiraBrik** [comment]
The moment with the button is not always reproduced. If I notice it again,
I'll record it on video. What do you recommend with ctrl+z? And then I'm
already thinking, should I create my own website, embed your script and
customize it for myself)

вс, 5 февр. 2023 г., 01:12 David Luzar ***@***.***>:

**Comment by sandstone991** [comment]
It doesn't work on arabic layout too, I think this probably has to do with using `event.key` instead of `event.code` somewhere in the code.

**Comment by LiraBrik** [comment]
Yes, It doesn't work on non latin layout. Event.code and event.key do not allow to create a universal implementation. A countless number of layouts divorced in the world ))  It is written on the Internet that the event.key is the preferred method. This is exactly what is implemented in Excalidrow. The solution is localization. I'm not a developer. Excalidraw is an open source program. Maybe someone will come up with something. I don't understand yet how to implement localization in the case of the keyboard. Although the localization of the interface is present, as we can see.
